packages/experimental/Badge/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@fluentui-react-native/badge",
+  "version": "0.0.1",
+  "description": "A cross-platform Badge component using the Fluent Design System. A badge is an additional visual discriptor for UI elements.",
+  "license": "MIT",
+  "author": "Microsoft <fluentuinativeowners@microsoft.com>",
+  "homepage": "https://github.com/microsoft/fluentui-react-native",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "lint": "fluentui-scripts eslint",
+    "depcheck": "fluentui-scripts depcheck",
+    "test": "fluentui-scripts jest",
+    "update-snapshots": "fluentui-scripts jest -u",
+    "prettier": "fluentui-scripts prettier",
+    "prettier-fix": "fluentui-scripts prettier --fix true"
+  },
+  "dependencies": {
+    "@fluentui-react-native/adapters": "^0.8.2",
+    "@fluentui-react-native/experimental-text": "^0.7.9",
+    "@fluentui-react-native/framework": "0.7.0",
```

Comment:
The framework package has been updated to 0,7.11 in master. You could do something like  "@fluentui-react-native/framework": ">=0.7.11 <1.0.0",

Reply:
I think this is what's causing the extra lines to show up in yarn.lock